core/client/tests/acceptance/settings/navigation-test.js

```diff
@@ -91,14 +129,105 @@ describe('Acceptance: Settings - Navigation', function () {
             triggerEvent('.gh-blognav-label:first input', 'blur');
 
             andThen(function () {
-                expect($('.gh-blognav-label:first input').val()).to.equal('Test');
+                expect(find('.gh-blognav-label:first input').val()).to.equal('Test');
             });
 
             visit('/settings/code-injection');
             visit('/settings/navigation');
 
             andThen(function () {
-                expect($('.gh-blognav-label:first input').val()).to.equal('Home');
+                expect(find('.gh-blognav-label:first input').val()).to.equal('Home');
+            });
+        });
+
+        it('can add and remove items', function (done) {
+            visit('/settings/navigation');
+
+            click('.gh-blognav-add');
+
+            andThen(function () {
+                expect(
+                    find('.gh-blognav-label:last .response').is(':visible'),
+                    'blank label has validation error'
+                ).to.be.true;
+
+                expect(
+                    find('.gh-blognav-url:last .response').is(':visible'),
+                    'blank url has validation error'
+                ).to.be.true;
+            });
+
+            fillIn('.gh-blognav-label:last input', 'New');
+            triggerEvent('.gh-blognav-label:last input', 'keypress', {});
+
+            andThen(function () {
+                expect(
+                    find('.gh-blognav-label:last .response').is(':visible'),
+                    'label validation is visible after typing'
+                ).to.be.false;
+
+                expect(
+                    find('.gh-blognav-url:last .response').is(':visible'),
+                    'blank url still has validation error'
+                ).to.be.true;
+            });
+
+            fillIn('.gh-blognav-url:last input', '/new');
+            triggerEvent('.gh-blognav-url:last input', 'keypress');
+            triggerEvent('.gh-blognav-url:last input', 'blur');
+
+            andThen(function () {
+                expect(
```

Comment:
This is what's causing one of the errors in the client tests. Is this due to improper fake-placeholder handling?